### ISSUE TITLE
Intro and ad detection error

### ISSUE THREAD

**Issue by CodeKid2024** [danrahn/MarkerEditorForPlex#33]
Hi, I imagine this would be a quick fix, but I was testing the beta version, and I believe there is a problem with the intro and ad addition. Maybe whatever kind of marker plex is looking isn't being correctly added or was recently, but the credits seem to work fine with no problems. 

I am running Windows 11 23H2 and am not using an arm processor.

**Comment by CodeKid2024** [comment]
Same bug with the latest official version.

**Comment by CodeKid2024** [comment]
I'm not really sure what kind of information you're looking for, so let me know if you need to see anything from my settings.

**Comment by danrahn** [comment]
What specific error(s) are you seeing? Does Marker Editor itself give you any warnings or errors (either in the UI or the console that launches when you start it)? After shutting down the editor, does [Get Info](https://support.plex.tv/articles/201998867-investigate-media-information-and-formats/#toc-1) > View XML not show your added/edited markers?

Beyond that, what server version are you running? And I'm assuming the server is running on the same machine you're using to run this application?

**Comment by CodeKid2024** [comment]
No, no specific errors are shown, but in the xml the intro is not being added. Only the credits that plex recognized are there.
![image](https://github.com/user-attachments/assets/49184422-c587-4b45-b3fd-76c0f6310c51)
![image](https://github.com/user-attachments/assets/b4e758b9-e6e6-447a-82a8-2e71f9f8dd44)
I just updated my server, so I assume it is the latest version for Windows. v.1.41.3.9314-a0bfb8370

**Comment by danrahn** [comment]
No idea what's going on there unfortunately. If you stop/restart marker editor, does the intro still show up? Anecdotally, I had no issues adding an intro marker with the latest 2.8.0-beta.2 on my Windows 10 server with PMS 1.41.3.9314. With verbose logging enabled in Marker Editor, the server console should show something like this:

```log
[2024.12.22 19:09:37.052][VERBOSE] ServerCore      - (192.168.1.2) POST: /add?metadataId=504764&start=0&end=66000&type=intro&final=0 
ConsoleLog.js:363
[2024.12.22 19:09:37.090][VERBOSE] PlexDB          - reindex: Reindexing 1 markers. 
ConsoleLog.js:363
[2024.12.22 19:09:37.200][VERBOSE] MarkerBackup    - 1 marker add(s) added to backup. 
ConsoleLog.js:363
[2024.12.22 19:09:37.200][INFO   ] CoreCommands    - Added intro marker to item 504764 [0-66000] 
ConsoleLog.js:363
[2024.12.22 19:09:37.200][VERBOSE] ServerHelpers   - Success: true
```

**Comment by CodeKid2024** [comment]
Yes, the intro does still show up after I tried setting it to verbose logging, but it didn't respond. It just stayed at info logging even after I restarted my server and closed the marker editor.. Does the file need to be placed in a certain directory on my pc, because currently I just have it in the downloads folder.

**Comment by danrahn** [comment]
It shouldn't have to be placed anywhere specific, but it would be good to at least have it in its own folder if it isn't already, so the layout would be something like this after going through the initial setup/startup:

```text
C:\Documents\MarkerEditor
    Backup\
      markerActions.db
    node_modules\
    config.json
    MarkerEditor.exe
    README.txt
```

If changing the server settings also doesn't seem to be working, there might be a bigger problem that I haven't been able to reproduce on either 2.7.1 or 2.8.0-beta.2. If you start completely fresh (e.g. redownload the zip file and extract it to a different location), does it still happen? I'd also double-check that Plex is picking up the right database - the console should have an INFO message along the lines of "Verifying database C:\Path\To\Database.db". To remove any other variables, also make sure Plex isn't running when trying to add the intro.
